### PR TITLE
[13.0][IMP] stock_picking_manual_package: Allow select in wizard the product to put in pack

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

ping @chienandalu @victoralmau

### CODE REVIEW

File: stock_picking_manual_package/wizards/stock_picking_manual_package_wiz.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright 2022 Sergio Teruel - Tecnativa
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class StockPicking(models.TransientModel):
```
Comment (rousseldenis) [inline stock_picking_manual_package/wizards/stock_picking_manual_package_wiz.py]: I'd suggest you to adapt this to model name

File: stock_picking_manual_package/wizards/stock_picking_manual_package_wizard_view.xml [hunk 2]
```diff
@@ -11,6 +11,13 @@
                 </group>
                 <p>Set how many lines you want to 'Put in pack'</p>
                 <group>
+                    <field name="picking_product_ids" invisible="1" />
```
Comment (rousseldenis) [inline stock_picking_manual_package/wizards/stock_picking_manual_package_wizard_view.xml:16]: I'd suggest you to enhance the `<p>` message in order to help easily user to deal with product_ids field (without hover).

**Comment by sergio-teruel** [comment]
Thanks @rousseldenis Changes done.. but I change the title of this PR because I have an idea in my mind... 
Maybe I could add a One2many field in the wizard to specifi lines by products... but by the moment is an idea...

**Review by victoralmau — APPROVED** [review]
Code review OK.

File: stock_picking_manual_package/wizards/stock_picking_manual_package_wiz.py [hunk 3]
```diff
@@ -15,17 +15,31 @@ class StockPicking(models.TransientModel):
         "all done quantities "
         "will be include in this package",
     )
+    product_ids = fields.Many2many(
+        comodel_name="product.product",
+        string="Products",
+        help="Select which products to include in the package",
+    )
+    picking_product_ids = fields.Many2many(
```
Comment (cesar-tecnativa) [inline stock_picking_manual_package/wizards/stock_picking_manual_package_wiz.py:23]: If you add two many2many product_product, you have to define table names.

**Review by cesar-tecnativa — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Review by chienandalu — APPROVED** [review]

**Pull request merged**